bdb/llmeta.c

```diff
@@ -2682,7 +2677,7 @@ int bdb_add_dummy_llmeta(void)
         logmsg(LOGMSG_ERROR, "%s bdb_tran_commit rc: %d bdberr: %d\n", __func__, rc,
                 bdberr);
         tran = NULL;
-        goto fail;
+        return -1;
```

Comment:
I think this code was correct.  There's no parent transaction, it's safe to retry.  And there was no "abort after commit" bug.